OpenDreamClient/EntryPoint.cs

```diff
@@ -18,6 +20,18 @@ public sealed class EntryPoint : GameClient {
         [Dependency]
         private readonly IDreamResourceManager _dreamResource = default!;
 
+        private const string UserAgent =
+            "Mozilla/4.0 (compatible; MSIE 7.0; Windows NT 6.2; WOW64; Trident/7.0; .NET4.0C; .NET4.0E; .NET CLR 2.0.50727; .NET CLR 3.0.30729; .NET CLR 3.5.30729) ";
```

Comment:
```suggestion
            "Mozilla/4.0 (compatible; MSIE 7.0; Windows NT 6.2; WOW64; Trident/7.0; .NET4.0C; .NET4.0E; .NET CLR 2.0.50727; .NET CLR 3.0.30729; .NET CLR 3.5.30729)";
```
tiny nitpick :)

Reply:
Fixed in [`edf5449` (#1129)](https://github.com/OpenDreamProject/OpenDream/pull/1129/commits/edf54492990893bfe097fc0196f748b0f8f5e407)